### ISSUE TITLE
Using closedXML in windows server 2012

### ISSUE THREAD

**Issue by SamirHasanov** [ClosedXML/ClosedXML#464]
**Do you want to request a *feature* or report a *bug*?**
- [ ] Bug
- [ ] Feature

**0.85**

e.g. 0.87.1

**Excel file is not parsed**

Hi. Maybe it is not the best place to ask this question, but couldn't find answer while googling. Apologies for that.

I am using closedXML 0.85 in my project to read data from xlsx files. Although in my local it works finely, I am struggling to get the same in the web server. I use win server 2012 to host the application. The references to closedXML and openXML are in place. 

I tried to catch exception but the application doesn't throw any error message. 

What do you think I am missing? 

Thanks in advance

**Comment by vbjay** [comment]
Logging of exceptions.   Check user permissions of the file folder.

**Comment by igitur** [comment]
You say it doesn't throw an exception? So what happens? What are the symptoms? 

And anyway, we need a minimal, verifiable and complete example of the code to help further. Start with a piece of code that just produces a blank workbook with 1 sheet and see if that works on the server. Gradually increase the complexity until you can reproduce the problem.

**Comment by SamirHasanov** [comment]
I just managed to solve the issue. 
Got to know that actually it had nothing wrong with closedXML. Originally, I am not only parsing excel data, but html also. So by relying on httppostedfilebase file.ContentType I was selecting excels and using closedXML methods afterwards. It appeared that win server doesn´t recognize contenttype properly and as a result closedXML method has never been called. It is something related to MIME types in IIS. In fact .xlsx extension is linked to application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, but application was using default MIME type / application/octet-stream. 
As a workaround I have installed MS Office on win server. Everything seems to be working for now. 
Thank you mates for promptness. Much appreciated.
